change/@azure-msal-browser-2020-11-10-16-13-10-idTokenHint.json

```diff
@@ -0,0 +1,8 @@
+{
+  "type": "none",
```

Comment:
I think this can be a patch version since it is adding functionality (although doesn't really matter as there will be other patch changes, I suppose).

Reply:
The browser changes are only doc updates all the code changes are in common